pom.xml

```diff
@@ -282,7 +363,168 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-site-plugin</artifactId>
+                <version>3.4</version>
+            </plugin>
         </plugins>
     </build>
 
+    <!-- =============================================================== -->
+    <!-- Gestion des rapports -->
```

Comment:
In English please :)

Reply:
Sorry, it was a hasty copy-paste. It is in english now